### PR TITLE
Backend volunteer field additions + profile picture endpoints

### PR DESCRIPTION
**Pull request by jansm04**
Updated the `volunteers` table to include include `pronouns`, `phone_number`, `city`, and `province`. 
- modified the `CREATE` and `INSERT` statements in the SQL DDL
- updated the `Volunteer` interface to include the new fields

Added a new `volunteer_profile_pics` table to store encoded images
- also added new `CREATE` and `INSERT` statements in the SQL DDL
- created a `ProfilePic` interface for request bodies
- implemented `POST`, `GET`, `PUT`, and `DELETE` endpoints for profile pictures

### CODE REVIEW

File: database/init/createTables.sql [hunk 1]
```diff
@@ -49,10 +50,21 @@ create table volunteers (
...
+    phone_number VARCHAR(15),
+    city VARCHAR(15),
+    province VARCHAR(15),
     FOREIGN KEY (fk_user_id) REFERENCES users(user_id)
     ON DELETE CASCADE
 );
 
+create table volunteer_profile_pics (
+    fk_volunteer_id VARCHAR(255) PRIMARY KEY,
+    profile_pic TEXT NOT NULL,
```
Comment (jjessieshang) [inline database/init/createTables.sql:63]: i'm assuming you plan on storing the binary representation of images in the database, in this case we can use a [BLOB](https://learn.microsoft.com/en-us/sql/relational-databases/blob/binary-large-object-blob-data-sql-server?view=sql-server-ver16) type field. Unless this is meant to be a url field. Let me know what you had in mind.

**Review by jjessieshang — COMMENTED** [review]
Thanks for making these changes thoroughly. Could you clarify what representation the images will be stored as in the database?

File: database/init/createTables.sql (hunk 1 above)
Comment (jansm04) [reply database/init/createTables.sql]: Yep sorry had it as text originally and forgot to change it. Just fixed it to blob

**Comment by jansm04** [comment]
Images will be stored using binary representations

**Comment by jansm04** [comment]
will hold off on this PR until frontend profile is finished

**Pull request closed without merging**